lib/utils.pm

```diff
@@ -849,31 +903,29 @@ the second run will update the system.
 =cut
 
 sub ssh_fully_patch_system {
-    my $remote = shift;
-    my $cmd_time = time();
-    my $resolver_option = get_var('PUBLIC_CLOUD_GEN_RESOLVER') ? '--debug-solver' : '';
-    my $cmd = "ssh $remote 'sudo zypper -n patch $resolver_option --with-interactive -l'";
-    # first run, possible update of packager -- exit code 103
-    my $ret = script_run($cmd, 1500);
-    record_info('zypper patch', 'The command zypper patch took ' . (time() - $cmd_time) . ' seconds.');
-    if ($ret != 0 && $ret != 102 && $ret != 103) {
-        if ($resolver_option) {
-            script_run("ssh $remote 'tar -czvf /tmp/solver.tar.gz /var/log/zypper.solverTestCase /var/log/zypper.log'");
-            script_run("scp $remote:/tmp/solver.tar.gz /tmp/solver.tar.gz");
-            upload_logs('/tmp/solver.tar.gz', failok => 1);
-        }
-        croak("Zypper failed with $ret");
-    }
-    $cmd_time = time();
-    # second run, full system update
-    $ret = script_run($cmd, 6000);
-    record_info('zypper patch', 'The second command zypper patch took ' . (time() - $cmd_time) . ' seconds.');
-    if ($resolver_option) {
-        script_run("ssh $remote 'tar -czvf /tmp/solver.tar.gz /var/log/zypper.solverTestCase /var/log/zypper.log'");
-        script_run("scp $remote:/tmp/solver.tar.gz /tmp/solver.tar.gz");
-        upload_logs('/tmp/solver.tar.gz', failok => 1);
-    }
-    croak("Zypper failed with $ret") if ($ret != 0 && $ret != 102);
+    my ($remote) = @_;
+    my $gen_resolver = get_var('PUBLIC_CLOUD_GEN_RESOLVER') ? 1 : 0;
+    my $debug_conflicts = get_var('PUBLIC_CLOUD_DEBUG_ZYPPER_CONFICTS') ? 1 : 0;
```

Comment:
if you would track this variable at the end it is not used anywhere

Reply:
ok I found where it is used ... nevertheless I think it is overkill . I mean why would you call `PUBLIC_CLOUD_GEN_RESOLVER=1` and `PUBLIC_CLOUD_DEBUG_ZYPPER_CONFICTS=0` ? IMHO if `PUBLIC_CLOUD_GEN_RESOLVER=1` it is enough